src/dmd/dinifile.d

```diff
@@ -177,7 +177,7 @@ void updateRealEnvironment(StringTable* environment)
  *      buffer = contents of configuration file
  *      sections = section names
  */
-void parseConfFile(StringTable* environment, const(char)* filename, const(char)* path, size_t length, ubyte* buffer, Strings* sections)
+void parseConfFile(StringTable* environment, const(char)* filename, const(char)* path, const size_t length, const(ubyte)* buffer, const(Strings)* sections)
```

Comment:
`const` on value type (`size_t`) is not useful nor desirable.

Reply:
Please explain why this is useful:

```d
void foo()
{
    const bar = 3;
}
```

But not this:

```d
void foo(const size_t bar) { }
```